docs/modeling/custom-proc.md

```
@@ -0,0 +1,46 @@
+procedure getUserFeeds(userId: Int, limit: Int?) : Post[] 
+```
+
+Mutation procedures (that write to the database) should be defined with `mutation procedure`:
+
+```zmodel title="schema.zmodel"
+mutation procedure signUp(email: String) : User
+```
+
+You can use all types supported by ZModel to define procedure parameters and return types, including:
+
+    - Primitive types like `Int`, `String`
```

markdownlint-cli2 (0.18.1)

Code block style
Expected: fenced; Actual: indented

(MD046, code-block-style)